magento/magento2
# After logging in customer is now not redirecting to Customer Dashboard by default

### Description
After logging in customer is now not redirecting to Customer Dashboard by default (based on https://github.com/magento/magento2/issues/10834#issuecomment-336713587).

### Manual testing scenarios
1. Open Admin -> "Stores" -> "Customers" -> "Customer Configuration" -> "Login Options" -> "Redirect Customer to Account Dashboard after Logging in" and make sure "Use system value" is checked.
2. On the Storefront create a Customer.
3. Log out if you are logged in.
4. Open CMS index page.  
5. Log in with Customer's credentials.
6. You should be redirected to CMS index page, not to Customer Dashboard.

### Contribution checklist
